backend/app/schemas/user.py

```diff
@@ -88,6 +88,25 @@ class Config:
         orm_mode = True
 
 
+class GetCurrentUserInfo(GetUserInfoNoRelation):
+    dept: GetAllDept | None = None
+    roles: list[GetAllRole]
+
+    @root_validator
+    def handel(cls, values):
+        """处理部门和角色"""
+        dept = values.get('dept')
+        if dept:
+            values['dept'] = dept.name
+        roles = values.get('roles')
+        if roles:
+            values['roles'] = [role.name for role in roles]
+        return values
+
+    class Config:
+        orm_mode = True
+
+
```

Comment:
If we don't need the associated information maybe we can change it directly in GetAllUserInfo?